src/main/java/com/hotels/bdp/circustrain/bigquery/conversion/BigQueryToHiveTableConverter.java

```diff
@@ -59,6 +59,7 @@ public BigQueryToHiveTableConverter() {
     Map<String, String> serDeParameters = new HashMap<>();
     serDeParameters.put("serialization.format", ",");
     serDeParameters.put("field.delim", ",");
+    serDeParameters.put("skip.header.line.count", "1");
```

Comment:
is this always the same? Should we make this configurable (we can go for this now and write a new issue if useful)? I'll approve in meantime

Reply:
This will always be the same (unless Google make a backwards incompatible change to their  BigQuery export api) I think this is good for the foreseeable future

Reply:
yup agree, thanks!